testData/pathfinding.data.js

```diff
@@ -0,0 +1,393 @@
+import { BuildingId } from "../src/types/main";
+
```

Comment:
Brief comments would be helpful for someone who hasn't worked on this. I know this is data, but is it an array of a defined type in our project?

Reply:
Yes, they are defined types. Will put comments to describe what is being stored in this file.

Reply:
yes, it's an array of lines, I will add comments to the data.